src/state-indicators.service.js

```diff
@@ -51,11 +53,15 @@ class StateIndicatorsService {
       const location = getLocation(lgas, states, stockCount)
       const locationThresholds = this.thresholdsService.calculateThresholds(location, stockCount)
       const stock = stockCount.stock
+      const products = this.productListService.relevant({byType: true})
```

Comment:
`this.productsListService.relevant` returns a promise, so you have to do the same thing as with `states` and `lgas` (see https://github.com/fielded/angular-nav-state-indicators/blob/master/src/state-indicators.service.js#L129,  https://github.com/fielded/angular-nav-state-indicators/blob/master/src/state-indicators.service.js#L118 and https://github.com/fielded/angular-nav-state-indicators/blob/master/src/state-indicators.service.js#L38)

Also, I think you want to call it with `{byType: true}`, with that option it will return and object like `dry: [..], frozen: [..]`, just call it without options and you should be fine.